src/System.Management.Automation/engine/lang/scriptblock.cs

```diff
@@ -443,7 +452,7 @@ public SteppablePipeline GetSteppablePipeline(CommandOrigin commandOrigin, objec
                 }
 
                 located = variablesToDefine.FirstOrDefault(
-                    v => string.Equals(v.Name, "_", StringComparison.OrdinalIgnoreCase));
+                    v => string.Equals(v.Name, "_", StringComparison.Ordinal));
```

Comment:
It is only one thing that we changed - perhaps current issue come from this. Let's revert.

Reply:
Checked the build locally, it _builds_ but something's definitely up. 

![image](https://user-images.githubusercontent.com/32407840/58384506-59dae480-7fb0-11e9-92b8-7cdb88c091b4.png)

Reverting this change doesn't fix whatever's broken. Gonna go over the changes again, see if there's something I overlooked.

Reply:
But previous commit passed CIs. Maybe PSReadline was updated?

Reply:
I think this was changed at the same time as I changed the other one, perhaps a bit too carelessly in this instance. 😕 

Everything appears to be working correctly, though!